python/ui/test/test_add_remove_elements.py

```diff
@@ -0,0 +1,41 @@
+from python.common.basebrowser import BrowserExample
+from python.ui.method.add_remove_page import AddRemove
+from python.ui.method.internet_home_page import Homepage
+
+
+class TestAddRemove:
+
+    def test_add_and_delete(self):
+        browser = BrowserExample().browser
+
+        the_homepage = Homepage(browser)
+        the_add_remove = AddRemove(browser)
```

Comment:
Small opinon/recommendation: I would name you page instantiated variable with "page" in it. like `the_add_remove_page` as it make is easier to understand that is what you are accessing later in the code.